dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -1150,6 +1150,10 @@ def convert_keys(keys)
 And(/^I press keys "([^"]*)" for element "([^"]*)"$/) do |key, selector|
   element = @browser.find_element(:css, selector)
   press_keys(element, key)
+  wait_short_until do
+    element_text = element.attribute("value")
+    element_text.include? key
```

Comment:
Are there remaining known cases in which this text might not match exactly?

Reply:
There were other tests that failed on the Drone build so I surmise that there are. I can follow up with an investigation to see if other tests can be updated similar to `personal_projects_gallery.feature`.

Reply:
https://drone.cdn-code.org/code-dot-org/code-dot-org/27907

Reply:
Seems reasonable.  It might be worth adding a brief comment to explain this deliberate behaviour.